docs/pages/aws-oss-guide.mdx

```diff
@@ -74,7 +74,7 @@ DynamoDB is a key-value and document database that delivers single-digit millise
 performance at any scale. For large clusters you can provision usage but for smaller
 deployments you can leverage DynamoDB's autoscaling.
 
-Teleport 4.0 leverages [DynamoDB's streaming feature](https://github.com/gravitational/teleport/issues/2430). When turning this on, you'll need
+Teleport 4.0+ leverages [DynamoDB's streaming feature](https://github.com/gravitational/teleport/issues/2430). When turning this on, you'll need
```

Comment:
```suggestion
Teleport leverages [DynamoDB's streaming feature](https://github.com/gravitational/teleport/issues/2430). When turning this on, you'll need
```
I would drop older references when the docs version are two major versions ahead.